msal/build.gradle

```diff
@@ -259,6 +258,7 @@ task sourcesJar(type: Jar) {
 }
 
 dependencies {
+    testImplementation project(path: ':common4j')
```

Comment:
hmm. we shouldn't need this :/

Reply:
yeah seems strange. We might be trying to call a common4j API directly in a test in msal and then we need this. Ideally we shouldn't do that and only call the Android common from msal